shell/platform/embedder/BUILD.gn

```diff
@@ -14,49 +14,68 @@ shell_gpu_configuration("embedder_gpu_configuration") {
   enable_metal = false
 }
 
-source_set("embedder") {
-  sources = [
-    "embedder.cc",
-    "embedder.h",
-    "embedder_engine.cc",
-    "embedder_engine.h",
-    "embedder_external_texture_gl.cc",
-    "embedder_external_texture_gl.h",
-    "embedder_include.c",
-    "embedder_platform_message_response.cc",
-    "embedder_platform_message_response.h",
-    "embedder_safe_access.h",
-    "embedder_surface.cc",
-    "embedder_surface.h",
-    "embedder_surface_gl.cc",
-    "embedder_surface_gl.h",
-    "embedder_surface_software.cc",
-    "embedder_surface_software.h",
-    "embedder_task_runner.cc",
-    "embedder_task_runner.h",
-    "embedder_thread_host.cc",
-    "embedder_thread_host.h",
-    "platform_view_embedder.cc",
-    "platform_view_embedder.h",
-    "vsync_waiter_embedder.cc",
-    "vsync_waiter_embedder.h",
-  ]
+# Template for the embedder build. Used to allow building it multiple times with
+# different flags.
+template("embedder_source_set") {
+  template_target_name = target_name
```

Comment:
This variable is redundant.

Reply:
Fixed. I misremembered issues I had with using target_name in a template target and thought this was needed (I think what I'm thinking of is needing access to this value *within* a sub-target, which isn't the case here.)